sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -288,16 +285,26 @@ class HadoopTableReader(
     }
   }
 
+  /**
+   * The entry of creating a RDD.
```

Comment:
Can you describe more about the motivation of this pr? I think this comment is a little meaningless....

Reply:
Yes, added more comment.